app/v1/models/models.py

```diff
@@ -0,0 +1,263 @@
+import datetime
+# =====================================DbUsers==============================================
```

Comment:
Get rid of this. Just put some description of what the class should do. For example

`## Create Users .....`